### ISSUE TITLE
Remove unnecessary ServerContextFactory class from Finger tutorial.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8628]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)| @Lukasa reported|
|-|-|
|Trac ID|trac#8628|
|Type|enhancement|
|Created|2016-07-19 12:41:12Z|
|Branch|https://github.com/twisted/twisted/tree/8628-lukasa-servercontextfactoryfinger|

There is now an unneeded `ServerContextFactory` class in the finger tutorial that should be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__8628 8628
type__enhancement enhancement
reporter__Lukasa Lukasa
priority__normal normal
milestone__None None
branch__8628_lukasa_servercontextfactoryfinger 8628-lukasa-servercontextfactoryfinger
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468932072947604 1468932072947604
changetime__1468955605271567 1468955605271567
version__None None
owner__Lukasa Lukasa

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_1"></a>|@Lukasa commented|
|-|-|

In changeset d1118b5678604c33816846710513f134f09e677c
```
#!CommitTicketReference repository="" revision="d1118b5678604c33816846710513f134f09e677c"
Add topfile for #8628
```

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa commented|
|-|-|

See https://github.com/twisted/twisted/pull/356

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set owner to @Lukasa|
|-|-|

See:
https://github.com/twisted/twisted/pull/356#issuecomment-233712722

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset 3621a92595eacc49924ea655f61d2c5847eef43d
```
#!CommitTicketReference repository="" revision="3621a92595eacc49924ea655f61d2c5847eef43d"
Merge 8628-lukasa-servercontextfactoryfinger: Remove unnecessary ServerContextFactory class.

Author: Lukasa
Reviewer: rodrigc
Fixes: #8628
```
